include/dev/scsi.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Based on FreeBSD `scsi_all.h`.
+ */
+#ifndef _SCSI_H_
```

Comment:
Definition name should reflect full path in mimiker's source tree starting from `include/`. In this case `_DEV_SCSI_H_`.

Reply:
Done.